flow/layers/container_layer.cc

```diff
@@ -49,6 +49,11 @@ void ContainerLayer::PaintChildren(PaintContext& context) const {
   }
 }
 
+float ContainerLayer::get_total_elevation() const {
```

Comment:
Sorry, I should have mentioned this in the earlier remark. OOL variable have CapitalCase.